e2e/rbd.go

```diff
@@ -674,14 +670,19 @@ var _ = Describe("RBD", func() {
 					e2elog.Failf("failed to poll: %v", err)
 				}
 
-				filePath := app.Spec.Containers[0].VolumeMounts[0].MountPath + "/test"
-				_, stdErr, err = execCommandInPod(
-					f,
-					fmt.Sprintf("echo 'Hello World' > %s", filePath),
-					app.Namespace,
-					&appOpt)
-				if err != nil || stdErr != "" {
-					e2elog.Failf("failed to write IO, err: %v, stdErr: %v ", err, stdErr)
+				// Writes on kernel < 5.4 are failing due to a bug in NBD driver,
+				// NBD zero cmd timeout handling is fixed with kernel >= 5.4
+				// see https://github.com/ceph/ceph-csi/issues/2204#issuecomment-930941047
+				if util.CheckKernelSupport(kernelRelease, nbdZeroIOtimeoutSupport) {
```

Comment:
Please report a warning message when the I/O testing is skipped.

Reply:
Done now, thanks!